### PR TITLE
Moved nametag code from cloudperms module to simplenametags plugin

### PR DESCRIPTION
**Pull request by mcmdev**
This pull request includes:

- [x] breaking changes
- [ ] no breaking changes

### Changes made to the repository:

Moved code for nametag handling from the CloudPerms Module into the SimpleNameTags Plugin.
As a result, the API-accessible updateNameTags methods were moved.
The corresponding example was also updated.

### Documentation of test results:

Changes have been tested on a v3.3.0 Environment, everything works as expected.

### CODE REVIEW

**Comment by derklaro** [comment]
Only pull requests targeting development are acceptable.

**Comment by byRoadrunner** [comment]
Thank you for your pull request @MCMDEV, before we can review it please rebase your changes to the development branch. 🙂

**Comment by mcmdev** [comment]
My bad, just changed that right now

**Comment by byRoadrunner** [comment]
> My bad, just changed that right now

Thanks, could you please also solve the merge conflicts?

File: cloudnet-plugins/cloudnet-simplenametags/src/main/java/de/dytanic/cloudnet/ext/simplenametags/CloudNetSimpleNameTagsPlugin.java [hunk 1]
```diff
@@ -21,4 +45,121 @@ public void onDisable() {
...
+
+        if (playerPermissionUser != null && playerPermissionGroup.get() == null) {
+            playerPermissionGroup.set(CloudNetDriver.getInstance().getPermissionManagement().getHighestPermissionGroup(playerPermissionUser));
+
+            if (playerPermissionGroup.get() == null) {
+                playerPermissionGroup.set(CloudNetDriver.getInstance().getPermissionManagement().getDefaultPermissionGroup());
+            }
+        }
+
+        int sortIdLength = CloudPermissionsManagement.getInstance().getGroups().stream()
```
Comment (0utplay) [inline cloudnet-plugins/cloudnet-simplenametags/src/main/java/de/dytanic/cloudnet/ext/simplenametags/CloudNetSimpleNameTagsPlugin.java]: The CloudPermissionManagement is deprecated, we should use CloudNetDriver#getPermissionManagement

File: cloudnet-plugins/cloudnet-simplenametags/src/main/java/de/dytanic/cloudnet/ext/simplenametags/CloudNetSimpleNameTagsPlugin.java [hunk 2]
```diff
@@ -21,4 +45,121 @@ public void onDisable() {
...
+                .map(IPermissionGroup::getSortId)
+                .map(String::valueOf)
+                .mapToInt(String::length)
+                .max()
+                .orElse(0);
+
+        initScoreboard(player);
+
+        Bukkit.getOnlinePlayers().forEach(all -> {
+            initScoreboard(all);
```
Comment (0utplay) [inline cloudnet-plugins/cloudnet-simplenametags/src/main/java/de/dytanic/cloudnet/ext/simplenametags/CloudNetSimpleNameTagsPlugin.java]: this.

File: cloudnet-plugins/cloudnet-simplenametags/src/main/java/de/dytanic/cloudnet/ext/simplenametags/CloudNetSimpleNameTagsPlugin.java [hunk 3]
```diff
@@ -21,4 +45,121 @@ public void onDisable() {
...
+        }
+
+        int sortIdLength = CloudPermissionsManagement.getInstance().getGroups().stream()
+                .map(IPermissionGroup::getSortId)
+                .map(String::valueOf)
+                .mapToInt(String::length)
+                .max()
+                .orElse(0);
+
+        initScoreboard(player);
```
Comment (0utplay) [inline cloudnet-plugins/cloudnet-simplenametags/src/main/java/de/dytanic/cloudnet/ext/simplenametags/CloudNetSimpleNameTagsPlugin.java]: this.

**Comment by 0utplay** [comment]
I think it would be easier if you copy the code from development into the simplenametags-plugin instead from the master?

**Comment by mcmdev** [comment]
Yeah, I guess I'll do that

**Comment by mcmdev** [comment]
This PR is going to become an absolute mess, so I'm going to close this one and reopen it from from the development branch.

**Pull request closed without merging**